### ISSUE TITLE
I am going to translate official chinese document into english.

### ISSUE THREAD

**Issue by netroby** [swoole/swoole-doc#5]

**Comment by netroby** [comment]
https://github.com/netroby/swoole-doc-en
The rendered html book online can be read via above url address.

The source code repository:
https://github.com/netroby/swoole-doc-en

**Comment by netroby** [comment]
@Atriedes Here we goes.

**Comment by netroby** [comment]
@matyhtf Is it possible to create an official repository ? i would like to contribute and maintained.

**Comment by Atriedes** [comment]
agreed we should separate english docs and chinese docs. Also Im gonna generate API documentation based on ide helper : https://github.com/Atriedes/swoole-ide-helper some of them already translated to english and added composer support.

**Comment by Atriedes** [comment]
@matyhtf what do you think?
